python/lsst/validate/drp/validate.py

```diff
@@ -137,6 +145,7 @@ def run(repo_or_json, metrics=None, makePrint=True, makePlot=True,
         if not os.path.isdir(repo_or_json):
             print("Could not find repo %s" % (repo_or_json))
             return
+        kwargs['metrics'] = metrics
```

Comment:
Under a slight expansion of the ticket to be "fix things that MWV broke in DM-11215 so that things run again", then yes.  I've expanded the ticket description to include this.

The general thing I was doing wrong was not correctly tracking which arguments were being passed through `**kwargs` but sometimes still needed to be inspected by the intervening functions, or which were being passed as explicit keywords to a function but not being passed along to functions called.

[Yes, this suggests some better design work is called for.  I'm somewhat saving that for adapting `validate_drp` into a true CommandLineTask DM-5096.]